packages/page-staking/src/Performance/FinalityCommittee.tsx

```diff
@@ -0,0 +1,42 @@
+// Copyright 2017-2022 @polkadot/app-staking authors & contributors
+// SPDX-License-Identifier: Apache-2.0
+import React from 'react';
+
+import { AddressSmall, CardSummary, SummaryBox, Table } from '@polkadot/react-components';
+
+import { useTranslation } from '../translate.js';
+import { useFinalityCommittee } from './useFinalityCommittee.js';
+
+type Props = {
+  session: number;
+};
+
+function FinalityCommittee ({ session }: Props) {
+  const { t } = useTranslation();
+  const finalityCommitteeAddresses = useFinalityCommittee(session);
+
+  const messageOnEmpty = finalityCommitteeAddresses && t("Data isn't available.");
```

Comment:
Is data really _not available_ or more like the _next finality committee_ or _validators_ are simply empty? (I'm guessing the former, cuase those lists can't be empty (?) but wanna make sure the message is not misleading)

Reply:
I think we have to have finality committee, otherwise we are stuck, but I think if we go far back enough the data might be unavailable. Theoretically shouldn't happen, because we limit how far back you can go

Reply:
Gotcha, so it's fine the way it is ✔️